### DISCUSSION TITLE
how to make header of <Fluent:RibbonGroupBox > has no space

### DISCUSSION THREAD

**Discussion by tomuGo** [fluentribbon/Fluent.Ribbon#1017] [Q&A] [answered]
how to hide header of <Fluent:RibbonGroupBox >
![image](https://user-images.githubusercontent.com/24789356/161422678-b191ec12-6832-499c-aa36-b54c7973c25b.png)

**Comment by batzen** [comment]
You can use the following code to make it consume no vertical space:
```
Header="{x:Null}"
HeaderTemplate="{x:Null}"
HeaderTemplateSelector="{x:Null}"
```

**Reply by tomuGo** [reply]
> You can use the following code to make it consume no vertical space:
> 
> ```
> Header="{x:Null}"
> HeaderTemplate="{x:Null}"
> HeaderTemplateSelector="{x:Null}"
> ```

thanksfor reply，but it does not work
![image](https://user-images.githubusercontent.com/24789356/162557305-7eb961c5-272b-47fa-a1df-66661fbf9369.png)

**Comment by batzen** [comment, marked as answer]
Well, hiding the header does not cause the buttons to get larger. If, for example, your buttons were set to medium size buttons from the second column would fit below.
If you want to increase the size of large buttons you'll have to overwrite the style and set your desired height there or set the height for them directly. But be aware that setting the height directly breaks dynamic sizing for those buttons.
